thinc/layers/dropout.py

```diff
@@ -9,16 +9,33 @@
 
 
 @registry.layers("Dropout.v1")
-def Dropout(rate: float = 0.0) -> Model[InT, InT]:
+def Dropout(
+    rate: float = 0.0, *, rate_attr_name: str = "dropout_rate"
+) -> Model[InT, InT]:
```

Comment:
I think we'll want to increment the version when introducing this type of change, i.e., `Dropout.v2`.

Reply:
I thought about that as well, but then decided against it because the change is backwards compatible: the new parameter has a default value and the behaviour remains the same if you use the default value, meaning that any code that calls the pre-existing version can continue to call this version and nothing will change. Or is that not what determines when the version number is incremented?

Reply:
The problem is that the filled config isn't backwards-compatible.